tools/cli/go-whisk-cli/commands/activation.go

```diff
@@ -31,12 +31,9 @@ import (
     "github.com/spf13/cobra"
 )
 
-const (
-    PollInterval = time.Second * 2
-    Delay        = time.Second * 5
-)
+const MAX_ACTIVATION_LIMIT = 200
```

Comment:
I agree, we wanted a quick fix right now. See the issue for the convo on dynamically providing max/min values.